practices/java-jdbc/main/com/uabc/dowa/jdbc/repository/StudentRepository.java

```diff
@@ -0,0 +1,133 @@
+package com.uabc.dowa.jdbc.repository;
+
+import com.uabc.dowa.jdbc.model.Student;
+import com.uabc.dowa.jdbc.connection.SmartConnection;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class StudentRepository {
+
+    public void persistStudent(Student student) throws SQLException {
+
+        String SQL = "INSERT INTO Student (firstName, lastName) VALUES( ?, ?)";
+
+        Connection conn = null;
+        PreparedStatement stm = null;
+
+        try {
+            conn = SmartConnection.getConnection();
+
+            stm = conn.prepareStatement(SQL);
+
+            stm.setString(1, student.getFirstName());
+            stm.setString(2, student.getLastName());
+
+            stm.executeUpdate();
+
+            System.out.println("Student saved...");
+
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
Ignore the exception seems like a bad idea, it's ok to hide the DB exception but at least throw a RuntimeException